Denik CZ.js

```diff
@@ -0,0 +1,131 @@
+function detectWeb(doc, url) {
+	return "newspaperArticle";
+}
+
+function doWeb(doc, url) {
+	scrape(doc, url);
+}
+
+function scrape(doc, url) {
+	var newArticle = new Zotero.Item('newspaperArticle');
+	newArticle.url = url;
+	newArticle.title = ZU.trimInternal(ZU.xpathText(doc, '//div[@class = "dv3-clanek-left-fix"]/h1')).replace(/^,/, "");
+	var date = ZU.xpathText(doc, '(//div[@class="dv3-clanek-datum"])');
+	var actualDateStr = '';
+	
+	if (ZU.trimInternal(date).indexOf('dnes') != -1) {
+		var actualDate = new Date(); 
+		actualDate.setHours(actualDate.getHours() + 2);
+		actualDateStr = actualDate.getUTCFullYear() + '-' +
+		('00' + (actualDate.getUTCMonth()+1)).slice(-2) + '-' +
+		('00' + actualDate.getUTCDate()).slice(-2);
+		date = actualDateStr;
+	} else if (ZU.trimInternal(date).indexOf('včera') != -1) {
+		var actualDate = new Date();
+		actualDate.setDate(actualDate.getDate() - 1);
+		actualDateStr = actualDate.getUTCFullYear() + '-' +
+		('00' + (actualDate.getUTCMonth()+1)).slice(-2) + '-' +
+		('00' + actualDate.getUTCDate()).slice(-2);
+		date = actualDateStr;
```

Comment:
The two block can be merged into one, just subtract an additional day if it's "včera"